### PR TITLE
Add Ubuntu 24.10 Oracular Oriole

### PR DESCRIPTION
**Pull request by Mailaender**
https://canonical.com/blog/canonical-releases-ubuntu-24-10-oracular-oriole

### CODE REVIEW

**Comment by karianna** [comment]
I don't _think_ we are supporting non LTS releases. labelling for PMC.

**Comment by Mailaender** [comment]
It says
https://github.com/adoptium/installer/blob/0b7c92ab49b962fbb1b1d01e3062ab4b67aaf9f7/linux/ca-certificates/debian/src/packageTest/java/org/adoptium/cacertificates/DebianFlavours.java#L34
and there are also remnants of the last STS version in the files.

**Comment by tellison** [comment]
Discussed in the PMC call today. @steelhead31 will provide updates.

**Comment by steelhead31** [comment]
I've taken a look at this, and will attempt to get this done this week. Im currently improving our release process for linux packages, and would like to get this merged before I complete my work, so that I can include this.  Once the checks are running green, we can merge this PR. An additional PR is needed to be able to publish the packages to artifactory for this new version, which I'll take care of as soon as Im able.

File: linux/Jenkinsfile [hunk 1]
```diff
@@ -523,19 +523,21 @@ def uploadDebArtifacts(String buildArch, String Version) {
...
         add more into list when available for release
         also update linux/{jdk,jre}/debian/main/packing/build.sh
     */
     def deb_versions = [
-            "trixie", // Debian/13
+            "trixie",   // Debian/13
             "bookworm", // Debian/12
             "bullseye", // Debian/11
             "buster",   // Debian/10
+            "oracular"  // Ubuntu/24.10 (STS)
```
Comment (steelhead31) [inline linux/Jenkinsfile:536]: Missing , at end of "oracular"

**Review by steelhead31 — CHANGES_REQUESTED** [review]

**Comment by steelhead31** [comment]
Im currently working on a PR that will fix the 2 failing actions JDK/JRE8 for alpine, the issue is not related to this PR, see : #1084

**Review by karianna — APPROVED** [review]

**Review by steelhead31 — APPROVED** [review]
Looks good.

**Comment by steelhead31** [comment]
PR for package republish is here : #1085

**Comment by steelhead31** [comment]
PR has been merged, I'll be building and pushing the packages up to artifactory throughout today.

**Comment by steelhead31** [comment]
DEBs for JDK8 have all been pushed.

**Pull request merged**